readme.markdown

```diff
@@ -261,6 +261,17 @@ To run tests a single time, simply run the following command. This is used in CI
 npm test
 ```
 
+# Troubleshooting
+
+### CSS Classes
+
+Dragula uses only four CSS classes. Their purpose is quickly explained below but you can check dist/dragula.css to see the corresponding CSS rules.
```

Comment:
Just a nitpick, use backticks for `dist/dragula.css`, and we should link it to the dist/dragula.css in the `master` branch